### PR TITLE
Enabling ImagePiiVerifyEngine to accept kwargs

### PR DESCRIPTION
**Pull request by niwilso**
## Change Description

Modifying `ImagePiiVerifyEngine` such that it can accept kwargs for a custom analyzer.

## Issue reference

This PR fixes issue #977 

## Checklist

- [X] I have reviewed the [contribution guidelines](https://github.com/microsoft/presidio/blob/main/CONTRIBUTING.md)
- [ ] I have signed the CLA (if required)
- [X] My code includes unit tests
- [X] All unit tests and lint checks pass locally
- [X] My PR contains documentation updates / additions if required

### CODE REVIEW

**Comment by niwilso** [comment]
Tests seem to be failing when installing dependencies. I did not change any packages or requirements in this PR. Would appreciate assistance in fixing this.

**Review by omri374 — APPROVED** [review]

File: presidio-image-redactor/presidio_image_redactor/image_pii_verify_engine.py [hunk 1]
```diff
@@ -19,24 +19,25 @@ def fig2img(fig):
 class ImagePiiVerifyEngine:
     """ImagePiiVerifyEngine class only supporting Pii verification currently."""
 
-    def __init__(self, analyzer_engine: Optional[ImageAnalyzerEngine] = None):
-        if not analyzer_engine:
-            analyzer_engine = ImageAnalyzerEngine()
-        self.analyzer_engine = analyzer_engine
+    def __init__(self, image_analyzer_engine: Optional[ImageAnalyzerEngine] = None):
```
Comment (SharonHart) [inline presidio-image-redactor/presidio_image_redactor/image_pii_verify_engine.py:22]:
```suggestion
    def __init__(self, image_analyzer_engine: Optional[ImageAnalyzerEngine] = ImageAnalyzerEngine()):
```

File: presidio-image-redactor/presidio_image_redactor/image_pii_verify_engine.py [hunk 2]
```diff
@@ -19,24 +19,25 @@ def fig2img(fig):
...
+        if not image_analyzer_engine:
+            image_analyzer_engine = ImageAnalyzerEngine()
+        self.image_analyzer_engine = image_analyzer_engine
```
Comment (SharonHart) [inline presidio-image-redactor/presidio_image_redactor/image_pii_verify_engine.py:25]:
```suggestion
    def __init__(self, image_analyzer_engine: Optional[ImageAnalyzerEngine] = ImageAnalyzerEngine()):
            self.image_analyzer_engine = image_analyzer_engine
```

**Review by SharonHart — APPROVED** [review]

File: presidio-image-redactor/presidio_image_redactor/image_pii_verify_engine.py (hunk 2 above)
Comment (omri374) [reply presidio-image-redactor/presidio_image_redactor/image_pii_verify_engine.py:25]: @SharonHart I wouldn't recommend this https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/#:~:text=In%20Python%2C%20when%20passing%20a,or%20even%20a%20class%20instance.

Comment (SharonHart) [reply presidio-image-redactor/presidio_image_redactor/image_pii_verify_engine.py:25]: Nice to know, fail enough :)

**Comment by omri374** [comment]
@niwilso there seems to be an issue with the CI. We're looking into this.

**Pull request merged**